src/Pos/Wallet/Web/Api.hs

```diff
@@ -77,52 +77,65 @@ type TestReset = API
     :> "reset"
     :> Post '[JSON] (Either WalletError ())
 
+-------------------------------------------------------------------------
+-- Wallet sets
+-------------------------------------------------------------------------
 
 type GetWalletSet = API
-    :> "walletSets"
+    :> "wallets"
```

Comment:
How about flipping `wallets` and `sets`, or remain just `sets`?
Again, I'm afraid of sudden clashes between `wallet` and `walletset` endpoints which may happen (in future)